<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <node name="imu" pkg="rr_platform" type="imu" output="screen">
        <param name="serial_port" value="/dev/ttyACM4"/>
    </node>
</launch>